src/components/Stepper/Stepper.js

```diff
@@ -18,13 +18,19 @@ export const Stepper = ({ steps }) => {
     return <span className={styles.Step}></span>
   }
 
-  const getTitleClassForStatus = (status) => {
-    if (status == 'complete') {
-      return styles.TitleComplete
-    } else if (status == 'active') {
-      return styles.TitleActive
+  const getTitle = (step) => {
+    if (step.status == 'complete' || step.status == 'active') {
+      return (
+        <Caption2.Regular400 color={COLORS.GRAY_PRIMARY}>
+          {step.title}
+        </Caption2.Regular400>
+      )
     }
-    return styles.Title
+    return (
+      <Caption2.Regular400 color={COLORS.GRAY_STROKE_AND_DISABLED}>
```

Comment:
minor non blocking nit - could set color in a variable and reduce this to just `<Caption2.Regular400 color={colorVar}>` if everything else is the same.